core/App/screens/PinEnter.tsx

```diff
@@ -310,8 +312,11 @@ const PinEnter: React.FC<PinEnterProps> = ({ setAuthenticated, pinEntryUsage = P
           onPress={() => {
             Keyboard.dismiss()
             onPinInputCompleted(pin)
+            setButtonLoading(true)
           }}
-        />
+        >
+          {buttonLoading && <ButtonLoading />}
+        </Button>
```

Comment:
If the user misses the PIN, the spinner never seems to be reset to false.

Reply:
good catch, I'll fix it momentarily